declarations_site/catalog/views.py

```diff
@@ -96,8 +98,6 @@ def search(request):
         search = base_search.query(
             "multi_match",
             query=query,
-            # Not ideal yet, as some of those fields aren't analyzed
-            type="cross_fields",
```

Comment:
I couldn't notice a decrease in results quality. Do you have a particular sample query, which would make use of this?